quintessence/AllegroFlare/TransitionFX/Base.q.yml

```diff
@@ -0,0 +1,89 @@
+properties:
+
+
+  - name: pasteboard_a
+    type: ALLEGRO_BITMAP*
+    init_with: nullptr
+    getter: explicit
+
+  - name: pasteboard_b
+    type: ALLEGRO_BITMAP*
+    init_with: nullptr
+    getter: explicit
+
+  - name: duration_sec
+    type: float
+    init_with: 1.0f
+    constructor_arg: true
+
+  - name: position
+    type: float
+    init_with: 0.0f
+
+  - name: finished
+    type: bool
+    init_with: false
+    getter: true
+
+  - name: initialized
+    type: bool
+    init_with: false
+
+
+functions:
+
+
+  - name: initialize
+    guards: [ (!initialized) ]
+    body: |
+      ALLEGRO_BITMAP *backbuffer = al_get_backbuffer(al_get_current_display());
+
+      pasteboard_a = al_create_bitmap(al_get_bitmap_width(backbuffer), al_get_bitmap_height(backbuffer));
+      pasteboard_b = al_create_bitmap(al_get_bitmap_width(backbuffer), al_get_bitmap_height(backbuffer));
```

Comment:
Most likely, will prefer to use an `AllegoFlare::RenderSurfaces::*` that is not a backbuffer for `pasteboard_a` and `pasteboard_b`.